res/controllers/Reloop-Digital-Jockey-2-IE.midi.xml

```diff
@@ -118,12 +118,12 @@
                 </options>
             </control>
             <control>   <!-- shifted -->
-                <group>[Channel1]</group>
-                <key>RDJ2.leftDeck.bendPlusButton.input</key>
+                <group>[Library]</group>
+                <key>GoToItem</key>

```

Comment:
If this mapping was more important (more recent controller) I'd insist to keep all library controls closer together (since the intended mapping / labels doesn't makes to me).
But it isn't, so that's okay I guess.

Did you look into the longpress function for Trax press? Tbh I'd still prefer the less used `maximize_library` control to be mapped there.
Though, if you want to finish this mapping quickly, maybe @alpham8 has motivation to implement that later on.

Reply:
Why do you think it does not make sense? It aligns with the labeling on the controller and manufacturer's intention. Normally, rather than twisting the small TRAX knob (which, as I already mentioned, feels cheap), you would turn the jog wheel to browse the library and push the **nearby** bend plus/minus buttons. This is really convenient. One hand on the shift button, the other hand operates the jog wheel and the buttons. Please take a look at the layout:
https://www.reloop.com/media/catalog/product/2/2/4/224092_Reloop_TP.jpg

I considered the longpress for Trax, but this would complicate current implementation and I would need to rewrite and retest a substantial part of the code. And more importantly, again, bend buttons are closer to the jog wheel than the TRAX knob, so it is really more convenient. And you don't need to open the manual after a year to recall how to operate the library.

Reply:
> Why do you think it does not make sense? It aligns with the labeling on the controller and manufacturer's intention.

I question the labels and the intention ;)
It's just that IMO (from my experience with the Reloop TerminalMix2/4) 4 controls are sufficient to navigate all library widgets (move, scroll, switch panes, select, collapse/expand items, clear search...):
* the `Trax` knob (turn + shift.turn + press + shift.press) (+ pressed.turn + shift.pressed.turn if you're adventurous)
* plus one button `X` (Back`` on TM4)
* 2 or more `Load` buttons

It seems to me Reloop tried to compensate the cheap Trax knob and the missing `X` button by using the wheel for scrolling. Also they designed the inertaction for other apps than Mixxx.
The result is that library controls are in two places: [Trax & Load A/B] and [wheel & **+-**] and +- do the same.
That's my concern. Though, I'm not motivated to work out alternatives with you if you're happy with how the current mapping fits your workflow.